src/language.ts

```diff
@@ -70,9 +86,11 @@ export const getChangeLanguageWebChat = (manager: FlexWebChat.Manager, config: C
         ...languageTranslations,
         ...configTranslations[language],
       });
+      setMainHeaderTitle(manager, language);
     } else {
       setConfigLanguage(defaultLanguage);
       setNewStrings({ ...twilioStrings, ...defaultLanguageTranslations, ...configTranslations[defaultLanguage] });
+      setMainHeaderTitle(manager, language);
```

Comment:
@acedeywin It should use `defaultLanguage` in the else block, instead of language

Reply:
@murilovmachado - It's done.